apps/console/src/features/organizations/components/organization-profile.tsx

```diff
@@ -0,0 +1,400 @@
+                                            readOnly={ true }
+                                            value={ organization.type === ORGANIZATION_TYPE.STRUCTURAL
+                                                ? "Structural"
+                                                : "Tenant"
+                                            }
+                                            ariaLabel={ "Organization Type" }
+                                            inputType={ "name" }
+                                            maxLength={ 32 }
+                                            minLength={ 3 }
+                                        />
+                                    )
+                                }
+                                {
+                                    organization?.created && (
+                                        <Field.Input
+                                            data-testid={ `${testId}-profile-form-created-input` }
+                                            name="created"
+                                            label={ t("Created") }
+                                            required={ false }
+                                            requiredErrorMessage=""
+                                            type="text"
+                                            readOnly={ true }
+                                            value={ moment(organization.created).format("YYYY-MM-DD hh:mm:ss") }
+                                            ariaLabel={ "Created" }
+                                            inputType={ "default" }
+                                            maxLength={ 32 }
+                                            minLength={ 3 }
+                                        />
+                                    )
+                                }
+                                {
+                                    organization?.lastModified && (
+                                        <Field.Input
+                                            data-testid={ `${testId}-profile-form-last-modified-input` }
+                                            name="lastModified"
+                                            label={ t("Last Modified") }
+                                            required={ false }
+                                            requiredErrorMessage=""
+                                            type="text"
+                                            readOnly={ true }
+                                            value={ moment(organization.lastModified).format("YYYY-MM-DD hh:mm:ss") }
+                                            ariaLabel={ "Last Modified" }
+                                            inputType={ "default" }
+                                            maxLength={ 32 }
+                                            minLength={ 3 }
+                                        />
+                                    )
```

Comment:
Should this be editable?

Reply:
No. Added ReadOnly=true to make it non-editable